providers/src/airflow/providers/google/cloud/hooks/vertex_ai/model_service.py

```diff
@@ -233,7 +233,7 @@ def upload_model(
         }
 
         if parent_model:
-            request["parent_model"] = parent_model
+            request["parent_model"] = f"projects/{project_id}/locations/{region}/models/{parent_model}"
```

Comment:
@VladaZakharova WDYT?
```suggestion
            request["parent_model"] = client.model_path(project_id, region, parent_model)
```